src/pymovements/events/processing.py

```diff
@@ -97,24 +90,22 @@ def process(self, events: EventDataFrame) -> pl.DataFrame:
 class EventGazeProcessor:
     """Processes event and gaze dataframes.
 
-    Attributes
+    Parameters
```

Comment:
Shouldn't parameters come here as an addition?

event_properties refers to both the Parameter for the initialization, and the Attribute of the instance.

Reply:
yes it should -- will fix it